src/tests/ftest/launch.py

```diff
@@ -1469,7 +1469,7 @@ def process_the_cores(avocado_logs_dir, test_yaml, args):
         return False
 
     # here because it's installed in install_debuginfos()
-    import magic    # pylint: disable=import-outside-toplevel
+    import magic    # pylint: disable=import-outside-toplevel,pylint-import-error
```

Comment:
Sorry I think I had a typo on my original suggestion, try this:
```suggestion
    import magic    # pylint: disable=import-outside-toplevel,import-error
```